### PR TITLE
Add CFC 3 article

### PR DESCRIPTION
**Pull request by fajar13k**
Start CFC 3 Wiki Article on Group Stage state

---

### CODE REVIEW

File: wiki/Tournaments/CFC/3/en.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+---
+tags:
+- Catch French Cup 3
+- CFC3
+---
+# Catch French Cup 3
+
+![CFC 3 Logo](img/logo.png)
+
+The **Catch French Cup 3 (***CFC 3***)** is a one-on-one french osu!catch tournament hosted by ![][flag_FR] [Kasumii-sama](/users/6177263). It is the 3rd installment of the Catch French Cup.
```
Comment (TPGPL) [inline wiki/Tournaments/CFC/3/en.md]: French

File: wiki/Tournaments/CFC/3/en.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
+---
+tags:
+- Catch French Cup 3
+- CFC3
+---
+# Catch French Cup 3
+
+![CFC 3 Logo](img/logo.png)
+
+The **Catch French Cup 3 (***CFC 3***)** is a one-on-one French osu!catch tournament hosted by ![][flag_FR] [Kasumii-sama](/users/6177263). It is the 3rd installment of the Catch French Cup.
```
Comment (TPGPL) [inline wiki/Tournaments/CFC/3/en.md]: `**Catch French Cup 3 (***CFC 3***)**` -> `**Catch French Cup 3** (***CFC 3***)`

File: wiki/Tournaments/CFC/3/en.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 10 months of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 6 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 4 months of osu!supporter tag |
+| *4th place* | 2 months of osu!supporter tag |
+
+## Organisation
+
+The Catch French Cup 3 is run by various community members.
```
Comment (TPGPL) [inline wiki/Tournaments/CFC/3/en.md]: The Catch French Cup 3 is run by various community members by distributing the multitude of tasks into various fields of responsibility.

File: wiki/Tournaments/CFC/3/en.md (hunk 2 above)
Comment (Joehuu) [inline wiki/Tournaments/CFC/3/en.md]: `**Catch French Cup 3 (***CFC 3***)**` -> `**Catch French Cup 3** (***CFC 3***)`

File: wiki/Tournaments/CFC/3/en.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+![CFC 3 Logo](img/logo.png)
+
+The **Catch French Cup 3 (***CFC 3***)** is a one-on-one French osu!catch tournament hosted by ![][flag_FR] [Kasumii-sama](/users/6177263). It is the 3rd installment of the Catch French Cup.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-08-15/2018-09-02 |
+| Live Drawings | 2018-09-01 (19:00 UTC) |
```
Comment (Joehuu) [inline wiki/Tournaments/CFC/3/en.md]: `Live Drawings` happen before `Registration Phase` end?

File: wiki/Tournaments/CFC/3/en.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+
+### Match Instructions
+
+1. The referee will create the room and invite the players in.
+2. The players will roll to determine the order of picking warmups and beatmaps.
+3. The higher roller will pick the first warmup beatmap.
+4. The lower roller will ban first, the higher roller will pick first.
+   - FreeMod is forced mod, meaning every player has to use at least one mod.
+   - Tiebreaker is not forced mod but you may choose a mod.
+   - Eligible mods for FreeMod and Tiebreaker are Easy, Flashlight, Hard Rock and Hidden.
```
Comment (Joehuu) [inline wiki/Tournaments/CFC/3/en.md]: Consistent spacing on the mods (i.e. `Hard Rock` or `HardRock`).

File: wiki/Tournaments/CFC/3/en.md (hunk 4 above)
Comment (fajar13k) [reply wiki/Tournaments/CFC/3/en.md]: It happened before Registration Phase indeed according to the Organiser

File: wiki/Tournaments/CFC/3/en.md [hunk 6]
```diff
@@ -0,0 +1,201 @@
+---
+tags:
+- Catch French Cup 3
+- CFC3
+---
```
Comment (TPGPL) [inline wiki/Tournaments/CFC/3/en.md]: add `CFC 3` as well

**Pull request merged**